src/openms/include/OpenMS/KERNEL/MSSpectrum.h

```diff
@@ -70,6 +68,7 @@ namespace OpenMS
     public RangeManager<1>,
     public SpectrumSettings
   {
+     friend class MSSpectrumHelper;
```

Comment:
nearly all were removed in https://github.com/OpenMS/OpenMS/pull/2472/files only a handful left